compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/SkikoUIView.kt

```diff
@@ -555,7 +577,12 @@ internal class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol {
             position: UITextPosition,
             toBoundary: UITextGranularity,
             inDirection: UITextDirection
-        ): UITextPosition? = null
+        ): UITextPosition? {
+            return null
+            if (position !is IntermediateTextPosition) {
```

Comment:
The code after return will be never executed. There are a few similar places in this file

Reply:
Yes. It is intented. Currently this function doesn't have implementation and returns just a stub. When we will write implementation we should check position argument.